packages/main/src/RPA/HTTP.py

```diff
@@ -258,3 +259,52 @@ def download(
         self._create_or_overwrite_target_file(dirname / filename, response, overwrite)
 
         return response
+
+    def check_vulnerabilities(self) -> List:
+        """Check for possible vulnerabilities in the installed runtime
+        environment packages.
+
+        Currently will check only for OpenSSL version and outputs warning message on any
+        discovered vulnerability.
+
+        :return: list of all check results
+
+        .. code-block:: robotframework
+
+            *** Tasks ***
+            Vulnerability Check
+                ${results}=    Check Vulnerabilities
+                FOR    ${result}    IN    @{results}
+                    Log To Console    TYPE: ${result}[type]
+                    Log To Console    VULNERABLE: ${result}[vulnerable]
+                    Log To Console    MESSAGE: ${result}[message]
+                END
+        """
+        all_results = []
+        vulnerable, message = self._check_openssl_vulnerabilities()
+        all_results.append(
+            {"type": "OpenSSL", "vulnerable": vulnerable, "message": message}
+        )
+        if vulnerable:
+            self.logger.warning(message)
+        return all_results
+
+    def _check_openssl_vulnerabilities(self):
+        message = "No OpenSSL detected"
+        try:
+            import ssl  # pylint: disable=C0415
```

Comment:
I think this means that we only check openssl version if pyOpenSSL is also present. I think it makes sense, but I wonder if there is some way to document it? But if this does not seem important to other reviewers I think it's fine to leave as is.